addons/l10n_bh/data/template/account.account-bh.csv

```diff
@@ -0,0 +1,138 @@
+"id","name","code","account_type","reconcile","name@ar_001"
```

Comment:
```suggestion
"id","name","code","account_type","reconcile","name@ar"
```
As you changed it to ar in the .po (which is the right move btw, as it's the iso code of Arabic)
And in other csv while you're at it